implementations/rust/ockam/ockam_api/src/nodes/models/forwarder.rs

```diff
@@ -106,3 +112,33 @@ impl<'a> From<RemoteForwarderInfo> for ForwarderInfo<'a> {
         }
     }
 }
+
+impl<'a> From<&ForwarderRegistryInfo> for ForwarderInfo<'a> {
+    fn from(inner: &ForwarderRegistryInfo) -> Self {
+        Self {
+            forwarding_route: inner.forwarding_route().to_string().into(),
+            remote_address: inner.remote_address().to_string().into(),
+            worker_address: inner.worker_address().to_string().into(),
+        }
+    }
+}
+
+/// Response body when returning a list of Forwarders
+#[derive(Debug, Clone, Decode, Encode)]
+#[rustfmt::skip]
+#[cbor(map)]
+pub struct ForwarderList<'a> {
```

Comment:
We have some mixed usages here, but I'm pretty sure you could return just the `Vec<ForwarderInfo<'a>>` as part of the CBOR body.

Reply:
I mirrored the `InletList` and `OutletList` structs used for the portals' `list` commands - perhaps we could refactor these out as well? They had the exact same structure. For now I've removed this in the upcoming latest commit